web-marketplace/src/graphql/indexer/IssuersByClassId.graphql

```diff
@@ -0,0 +1,9 @@
+query IndexerIssuersByClassId($classId: String) {
+  allClassIssuers(condition: { latest: true, classId: $classId }) {
+    nodes {
+      blockHeight
```

Comment:
Same here, it looks like this is unused. Any reason for including it?

Reply:
Same https://github.com/regen-network/regen-web/pull/2041#discussion_r1303876078